sunpy/map/map_factory.py

```diff
@@ -184,17 +186,23 @@ def _parse_args(self, *args, silence_errors=False, allow_errors=False, **kwargs)
             elif isinstance(arg, str) and is_url(arg):
                 # Replace URL string with a Request object to dispatch on later
                 args[i] = Request(arg)
-            elif possibly_a_path(arg):
+            elif possibly_a_path(arg) and not is_uri(arg):
                 # Replace path strings with Path objects
                 args[i] = pathlib.Path(arg)
+            elif is_uri(arg):
+                # Replace uri string with list of fsspec OpenFile objects
+                fsspec_kw = kwargs.get("fsspec_kwargs", {})
+                args[i] = fsspec.open_files(arg, **fsspec_kw)
             i += 1
 
         # Parse the arguments
         # Note that this list can also contain GenericMaps if they are directly given to the factory
         data_header_pairs = []
         for arg in args:
             try:
-                data_header_pairs += self._parse_arg(arg, silence_errors=silence_errors, allow_errors=allow_errors, **kwargs)
+                data_header_pairs += self._parse_arg(
+                    arg, silence_errors=silence_errors, allow_errors=allow_errors, **kwargs
+                )
```

Comment:
I am going to be a pain but could you undo all of these style changes in this and the timeseries file?

Reply:
changed